supertokens/main.go

```diff
@@ -59,12 +59,12 @@ func GetUserCount(includeRecipeIds *[]string) (float64, error) {
 	return getUserCount(includeRecipeIds)
 }
 
-func GetUsersOldestFirst(paginationToken *string, limit *int, includeRecipeIds *[]string) (UserPaginationResult, error) {
-	return getUsers("ASC", paginationToken, limit, includeRecipeIds)
+func GetUsersOldestFirst(paginationToken *string, limit *int, includeRecipeIds *[]string, queryParams map[string]string) (UserPaginationResult, error) {
+	return getUsers("ASC", paginationToken, limit, includeRecipeIds, queryParams)
 }
 
-func GetUsersNewestFirst(paginationToken *string, limit *int, includeRecipeIds *[]string) (UserPaginationResult, error) {
-	return getUsers("DESC", paginationToken, limit, includeRecipeIds)
+func GetUsersNewestFirst(paginationToken *string, limit *int, includeRecipeIds *[]string, queryParams map[string]string) (UserPaginationResult, error) {
+	return getUsers("DESC", paginationToken, limit, includeRecipeIds, queryParams)
 }
```

Comment:
Do not modify the signature of these functions, this will make it a breaking sdk version update. Instead make a variant of `getUsers` that takes query params optionally and call that from the existing getUsers function.

Then use the function with query params in the dashboard API